### PR TITLE
Add name of contributors

### PR DESCRIPTION
**Pull request by corlettim**
Adding names of contributors for the demo in the first lab

### CODE REVIEW

File: software_engineering.md [hunk 1]
```diff
@@ -5,4 +5,4 @@
 ## Signed (Add your name)
 
 JJ
-
+Megan Corletti
```
Comment (noorbuchi) [inline software_engineering.md:9]: Please don't forget my name too

**Review by noorbuchi — CHANGES_REQUESTED** [review]
Please add my name as well as other contributors

File: software_engineering.md (hunk 1 above)
Comment (corlettim) [reply software_engineering.md:9]: Just added it!

**Review by noorbuchi — APPROVED** [review]
Looks good! Ready to merge!

**Pull request merged**